src/ansys/fluent/core/session.py

```diff
@@ -175,13 +191,17 @@ class Fields:
 
             def __init__(self, _session):
                 """Initialize Fields."""
+                self.is_data_valid = (
```

Comment:
Is this getting evaluated here? We need to pass a callable to the service instances.

Reply:
@mkundu1 Yes, right. Done. Thank you.

```
>>> from ansys.fluent.core import launch_fluent
>>> solver = launch_fluent(ui_mode="gui")
>>> solver._app_utilities.is_solution_data_available()
False
>>> solver.fields._is_solution_data_valid()
False
>>> Fast-loading "D:\ANSYSDev\vNNN\fluent\fluent25.2\\addons\afd\lib\hdfio.bin"
Done.
Multicore SMT processors detected. Processor affinity set!

Reading from AAPQ13pUvWxQtxN:"D:/Repos/example-data/pyfluent/mixing_elbow/mixing_elbow.cas.h5" in NODE0 mode ...
  Reading mesh ...
       17822 cells,     1 cell zone  ...
          17822 mixed cells,  zone id: 87
       91581 faces,     7 face zones ...
           2168 polygonal wall faces,  zone id: 34
            268 polygonal wall faces,  zone id: 33
            155 polygonal pressure-outlet faces,  zone id: 32
            152 polygonal velocity-inlet faces,  zone id: 31
             55 polygonal velocity-inlet faces,  zone id: 30
           2001 polygonal symmetry faces,  zone id: 29
          86782 mixed interior faces,  zone id: 89
       66417 nodes,     1 node zone  ...
Warning: reading 2 partition grid onto 1 compute node machine.
         Combining every 2 partitions.
  Done.


Building...
     mesh
     materials,
     interface,
     domains,
        mixture
     zones,
        wall-elbow
        wall-inlet
        outlet
        cold-inlet
        hot-inlet
        symmetry-xyplane
        interior--elbow-fluid
        elbow-fluid
     parallel,
Done.

Reading from AAPQ13pUvWxQtxN:"D:/Repos/example-data/pyfluent/mixing_elbow/mixing_elbow.dat.h5" in NODE0 mode ...


  Reading results.
Parallel variables...
Done.

>>> solver._app_utilities.is_solution_data_available()
True
>>> solver.fields._is_solution_data_valid()
True
>>>
```